AsyncDisplayKit/ASTableView.h

```diff
@@ -303,6 +302,19 @@ NS_ASSUME_NONNULL_BEGIN
 @optional
 
 /**
+ * Similar to -tableView:nodeForRowAtIndexPath:.
+ *
+ * @param tableView The sender.
+ *
+ * @param indexPath The index path of the requested node.
+ *
+ * @returns a block that creates the node for display at this indexpath.  Must be thread-safe (can be called on the main thread or a background
+ * queue) and should not implement reuse (it will be called once per row).
+ */
+
+- (ASDataControllerCellNodeBlock)tableView:(ASTableView *)tableView nodeBlockForRowAtIndexPath:(NSIndexPath *)indexPath;
```

Comment:
Can we rename the block type to be a much simpler "ASCellNodeBlock", or do you have any thoughts on why it should be more specific? Sometimes the typeahead requires specifying the correct return type, and because data controller is internal implementation detail anyway, we should probably not introduced the terminology in the more public facing API